config/channels/in-memory-channel/500-controller.yaml

```diff
@@ -44,13 +45,12 @@ spec:
             valueFrom:
               fieldRef:
                 fieldPath: metadata.namespace
+
+        securityContext:
+          allowPrivilegeEscalation: false
+
         ports:
-          - containerPort: 9090
-            name: metrics
-        volumeMounts:
```

Comment:
Why's this dropped? is this not used?

Reply:
We use watch.  A bunch of them weren't cleaned up.